godotproject/Menus/InGameMenu/PauseSubmenu.gd

```diff
@@ -1,9 +1,9 @@
 extends MarginContainer
 
 export (NodePath) var first_focus_path
-onready var first_focus = get_node(first_focus_path)
+onready var first_focus = get_node(first_focus_path) if first_focus_path else null
```

Comment:
This type of change fixes some "errors" that can come up during game loading. It still works the same but this just cleans it up.